author/urls.py

```diff
@@ -1,7 +1,9 @@
 from django.urls import path
 from author.views import AuthorListView, AuthorDetailView
+from . import views
```

Comment:
In the previous line were all imports from views. So I think it would be better to import create_author there